---
title: 'Cost of short read RNA sequencing'
date: 2025-09-05
permalink: /posts/2025/09/short-read-rnaseq/
tags:
  - Experiment Costs
  - RNAseq
  - short-read
  - Data assets
toc: true
excerpt: 'RNA sequencing is performed to quantify the relative abundance of various RNA in a sample.'
---

## Why do you do this experiment?

Sequencing RNA enables the identification and quantification of RNA expressed in a cell or a sample (the transcriptome).

**Input** 100k-1M Live cells, FFPE, frozen cells or 25-250ng RNA

**Output** Fastq file (20-100M PE reads, 60-300Gb) -> Gene expression

## Strategic Value

- By comparing multiple samples, we know the effect of perturbations (drug, disease, [knock-out](/2025-09-02-single-ko.md), etc) on the transcriptome of the cell. This can be used to understand gene regulation, how a drug works, or which processes a disease affects.
- RNAseq provides the sequence of all expressed genes, meaning variants (e.g. SNPs, gene fusions) can be called but coverage will be biased towards highly expressed genes.
In the context of cancer and with deep enough RNAseq, sub-clonal exonic mutations can be detected for most genes.

## Cost & Scale

- Variable per run: **\\$150/sample**. Range: \\$118 - \\$236
- Cost breakdown:
    + RNA extraction: \\$56
    + Short read library preparation: \\$50
    + Sequencing (20-100M reads, 4-30Gb): \\$12-\\$120
- Capex: Thermocycler (\\$10-20k), TapeStation (\\$6-30k), NGS Sequencer (\\$50k-1M)

## Experimental Modules

1. RNA extraction (2h30, 40' hands-on)
2. Sequencing library preparation (6h, 2h hands-on)
3. Sequencing run (4-24h depending on the sequencer)

## Ops & Throughput

**Turnaround**: 3+ days (day 1 extraction, day 2 library prep, day 3 or later sequencing)

**Hands-on time**: 4h

**Parallelizability**: High. All steps can be done in parallel for as many samples as needed.

**Bottlenecks**: availability of Tapestation (16 lanes) and thermocycler (96 wells).

**Batching**: 1 to 16 samples per technician.

**Automation readiness**: Full, with commercial solutions available.

**Outsourceability**: Yes.

**Data scale**: 20-100M reads/sample, ~30Gb/sample

## Data API
Raw format: FASTQ

Processed format: count matrix

Resolution: gene-level expression, single nucleotide variant

## Analysis Ecosystem

1. QC and cleaning
    - [fastqc](https://www.bioinformatics.babraham.ac.uk/projects/fastqc/): Quality control of the run
    - [cutadapt](https://cutadapt.readthedocs.io/en/stable/): Trimming of sequencing adapters from the reads
2. Alignement:
    - [STAR aligner](https://hbctraining.github.io/Intro-to-rnaseq-hpc-O2/lessons/03_alignment.html)
    - [bowtie2](https://bowtie-bio.sourceforge.net/bowtie2/index.shtml) 
    - [kallisto](https://pachterlab.github.io/kallisto/about): Transcript quantification via pseudo-alignement
    - [Salmon](https://combine-lab.github.io/salmon/): Transcript quantification via quasi-alignement
3. Gene expression quantification:
    - [htseq-count](https://htseq.readthedocs.io/en/release_0.11.1/count.html): Gene-read overlap counts
4. Differential expression
    - [Sleuth](https://pachterlab.github.io/sleuth_walkthroughs/trapnell/analysis.html)
    - [DESeq2](https://bioconductor.org/packages/release/bioc/html/DESeq2.html) or [PyDESeq2](https://pydeseq2.readthedocs.io/en/stable/)
    <!-- - [glmgampoi](https://bioconductor.org/packages/release/bioc/html/glmGamPoi.html) -->
    - [edgeR](https://bioconductor.org/packages/release/bioc/html/edgeR.html) or [edgePy](https://edgepy.readthedocs.io/en/latest/index.html)

## Public datasets

- [The Cancer Genome Atlas (TCGA)](https://www.cancer.gov/ccg/research/genome-sequencing/tcga): RNAseq (2x50bp) and WES for more than 20k tumors
- [Genotype-Tissue Expression (GTEx)](https://gtexportal.org/home/): RNAseq from all major organs from >700 individuals
- [Gene Expression Omnibus (GEO)](https://www.ncbi.nlm.nih.gov/geo/): Repository of sequencing data from publications
- [European Nucleotide Archive (ENA)](https://www.ebi.ac.uk/ena/browser/home): Repository of sequencing data from publications
- [recount3](https://rna.recount.bio/): data from TCGA and GTEx reprossed with a uniform pipeline
See also [this list](https://bigomics.ch/blog/ultimate-guide-to-public-rnaseq-and-sc-rna-seq-databases/)

## Pitfalls & Failure Modes

- Don't skip the ribo-depletion or polyA enrichment step, they represent most of the extration cost but are there for a reason. [>90%](https://www.frontiersin.org/files/Articles/127231/fgene-06-00002-HTML/image_m/fgene-06-00002-t001.jpg)[^1] of RNA in a cell are rRNA or tRNA. Sequencing total RNA from a cell without size selection with short read sequencing would yield around 70% of rRNA reads and 15% of tRNA reads which are not very interesting populations (unless you look at base modifications, which is not done in short read). With the cheap cost of sequencing nowdays you should systematically go for ribo-depletion over polyA. Batch correction can integrate your ribo-depleted data with a polyA cohorts without problems.
- Most protocols for RNAseq are optimized for the extraction of RNA longer than 20bp and will size select the sequencing library to 300-500bp. This will exclude small RNA populations (tRNA, miRNA, snoRNA, etc). If you are interested in those populations use dedicated kit (e.g [Qiagen miRNAeasy](https://www.qiagen.com/us/product-categories/discovery-and-translational-research/dna-rna-purification/rna-purification/mirna)) and remove the size selection steps.

[^1] https://www.frontiersin.org/journals/genetics/articles/10.3389/fgene.2015.00002/full

## Related publications

- [NEB RNA protocol](https://www.neb.com/en/-/media/nebus/files/manuals/manuale7760_e7765-w-umi-rna-adaptors-e7416.pdf) (section 4)
- [Zhao2018](https://www.nature.com/articles/s41598-018-23226-4) compares the differences between RiboZero and polyA enrichment in term of exonic coverage and transcript diversity.

## Order list

Plenty of suppliers exist for this kind of protocol and you can mostly mix an match suppliers to your liking for each step. I used NEB as a convenient example as their documentation is quite clear.

|Item|Cost|Number of experiments|Link|
|---------|--------|--------|
||Monarch速 Total RNA Miniprep Kit|300|50https://www.neb.com/en/products/t2010-monarch-total-rna-miniprep-kit?srsltid=AfmBOopSZmPKF4Cfc-PLtnsJVH3Cw5xaUBpW1I56u-Zhhk1bdz_qEuKi [](30 minutes fully hands-on, $6/sample)|
|NEBNext速 rRNA Depletion Kit|1170|24|https://www.neb.com/en/products/e7400-nebnext-rrna-depletion-kit-v2-human-mouse-rat [](2h, 10' hands on, $50/sample)|
|NEBNext Ultra II Directional RNA Library Prep Kit Illumina|1100|24|https://www.neb.com/en/products/e7760-nebnext-ultra-ii-directional-rna-library-prep-kit-for-illumina?srsltid=AfmBOooPomu_ib-QTTzKump5qvf8Tz8iLRobH3FuSFLhvdkatczjhqMW [](6h, 2h hands-on, $45/sample)|
|NEBNext速 Multiplex Oligos for Illumina速|120|24|https://www.neb.com/en/products/e7335-nebnext-multiplex-oligos-for-illumina-index-primers-set-1 []($5/sample)|
|||||
|---------|--------|--------|
|Total per xp|\\$200|1||
|---------|--------|--------|

## Protocol variations

- RNA extraction should yield [10-30pg of RNA/cell](https://www.qiagen.com/us/resources/faq/2946)
- Ultra-low-input protocols based on direct reverse transcription enable RNAseq from as low as 10 cells input (e.g from [Thermo-Fischer](https://www.thermofisher.com/fr/fr/home/life-science/pcr/reverse-transcription/superscript-cellsdirect.html)).


{% include costs_disclaimer.html %}
